README.rst

```diff
@@ -3,6 +3,25 @@ CNXML Library
 
 This is a python library for working with CNXML content, which includes the CNXML and COLLXML formats.
 
+Install
+-------
+
+This library requires Python >=3.5.
+
+Install using::
+
+   pip install -e "."
+
+Tests
+-----
+
+Run the tests using::
+
+   python setup.py test
```

Comment:
I thought the python 3 executable is `python3` since python 2 is just `python`, no?